plugins/obs-outputs/net-if.c

```diff
@@ -134,6 +134,36 @@ static inline bool is_loopback(struct ifaddrs *ifa)
 	return n && (strcmp(n, "lo") == 0 || strcmp(n, "lo0") == 0);
 }
 
+void netif_get_ifaces(struct netif_siface_data *ifaces)
+{
+	da_init(ifaces->ifaces);
+
+	struct ifaddrs *ifaddr, *ifa;
+	unsigned int family;
+
+	if (getifaddrs(&ifaddr) == -1) {
+		warn("getifaddrs() failed");
+		return;
+	}
+
+	for (ifa = ifaddr; ifa != NULL; ifa = ifa->ifa_next) {
+		if (ifa->ifa_addr == NULL || is_loopback(ifa))
+			continue;
+		family = ifa->ifa_addr->sa_family;
+
+		if (family == AF_INET) {
```

Comment:
Before it was set to AF_PACKET, but for macOS and FreeBSD, I switched it to AF_INET.
It is a good idea ? Or should I switch back and try to use AF_LINK for macOS and FreeBSD ?
Because the way it is now, if the interface have no IPv4 address but have a IPv6 one, it will not be listed.

Reply:
Now I use AF_PACKET and AF_LINK